third_party/blink/renderer/platform/media/key_system_config_selector.cc

```diff
@@ -411,12 +418,24 @@ bool KeySystemConfigSelector::IsSupportedContentType(
   // is done primarily to validate extended codecs, but it also ensures that the
   // CDM cannot support codecs that Chrome does not (which could complicate the
   // robustness algorithm).
+#if BUILDFLAG(IS_COBALT)
+#if BUILDFLAG(USE_STARBOARD_MEDIA)
+  if (SbMediaCanPlayMimeAndKeySystem(container_mime_type.c_str(),
```

Comment:
We should ensure that the other path also logs.  Feel free to add a TODO if it should be addressed into SbDrm integration PR.